src/libraries/System.Runtime/tests/System.Runtime.Extensions.Tests/System/Convert.ToHexString.cs

```diff
@@ -14,13 +14,23 @@ public static void KnownByteSequence()
         {
             byte[] inputBytes = new byte[] { 0x00, 0x01, 0x02, 0xFD, 0xFE, 0xFF };
             Assert.Equal("000102FDFEFF", Convert.ToHexString(inputBytes));
+            
+            Span<char> output = new char[12];
```

Comment:
Nit: usually we don't use the expression `Span<T> variable = array`. Either declare the variable as array and implicit cast to span when using, or use `stackalloc` to get a span.

Reply:
> usually we don't use the expression Span<T> variable = array

I do not see a problem with it, in particular in tests. You can find many existing instances of it in tests.